V5 release note - Automation part (#871)
akeneo/pim-helpcenter: content/updates/automation/4-Enhanced-rule-management.md

# Enhanced rule management

## Quickly duplicate your rules
::: meta-data type="New" ee-only="true" link-to-doc="../articles/manage-your-rules.html#duplicate-a-rule"
:::

> Sometimes you can have very similar rules with just one different condition. In order to avoid rewriting the whole rule, you can now duplicate it in two clicks! What a time-saver, isn't it?

Reviewer: peut-être élargir l'exemple en parlant d'une différence simple sur les conditions et/ou les actions ? (pour pas que les users croient qu'on peut modifier uniquement les conditions)

Author: ```suggestion
Sometimes you can have very similar rules with just one different condition or action. In order to avoid rewriting the whole rule, you can now duplicate it in two clicks! What a time-saver, isn't it?
```

> Sometimes, you may have very similar rules with only one condition that is different. In order to avoid re-writing the whole rule, you are now able to duplicate it in two clicks! What a time saver, isn't it? :wink:

Reviewer 2: ```suggestion
Sometimes you can have very similar rules with just one different condition. In order to avoid rewriting the whole rule, you can now duplicate it in two clicks! What a time-saver, isn't it?
```

![Duplicate your rule](../img/Rules_Duplicate.png)

## Define a status for your rules
::: meta-data type="New" ee-only="true" link-to-doc="../articles/manage-your-rules.html#define-a-status-for-a-rule"
:::

> You can now easily activate or deactivate your rules. Very useful when the creation/edition of the rule is not finished yet! Setting its status to "disabled" will pause the automatic execution of the rule.  

Reviewer 2: ```suggestion
You can now easily enable or disable your rules. Very useful when the creation or edition of your rule is not finished yet! Setting its status to "disabled" will pause the automatic execution of the rule.  
```

Author: ```suggestion
You can now easily enable or disable your rules. Very useful when the creation or edition of your rule is not finished yet! Setting its status to `disabled` will pause the automatic execution of the rule. 
```

Once the rule is ready, you just have to enable it to let it run!

![Define a status for your rules](../img/Rules_Status.png)

> ## Name your rules

Reviewer 2: ```suggestion
## Localize the labels of your rules
```

Reviewer 2: Ok!

::: meta-data type="New" ee-only="true"
:::

> You can now name your rules with a clear label. You can also define a rule name per PIM locale enabled. The rule label is visible under your rules grid and under your smart attributes: if you have international teams you can now make sure that they understand the purpose of a rule.

Reviewer 3: ```suggestion
You can now name your rules with a clear label. You can also define a rule's name per PIM locale enabled. The rule's label is visible in your rules grid and in your smart attributes: if you have international teams you can now make sure that they understand the purpose of a rule.
```
